examples/repository_delete_branch_on_merge/README.md

```diff
@@ -0,0 +1,18 @@
+# Repository `delete_branch_on_merge` Example
+
+This displays configurability of the `delete_branch_on_merge` feature for GitHub repositories.
+
+This example will create a repository in the specified `owner` organization or a personal account. See https://www.terraform.io/docs/providers/github/index.html for details on configuring [`providers.tf`](./providers.tf) accordingly.
```

Comment:
for now, maybe remove` or a personal account` since the provider only supports org-owned repo's?

Reply:
```suggestion
This example will create a repository in the specified `owner` organization. See https://www.terraform.io/docs/providers/github/index.html for details on configuring [`providers.tf`](./providers.tf) accordingly.
```